elastalert/elastalert.py

```diff
@@ -152,6 +153,10 @@ def __init__(self, args):
         self.disabled_rules = []
         self.replace_dots_in_field_names = self.conf.get('replace_dots_in_field_names', False)
         self.string_multi_field_name = self.conf.get('string_multi_field_name', False)
+        self.statsd_prefix = self.conf.get('statsd_metrics_prefix', '')
```

Comment:
no need to persis this on the object since it is not used anywhere other than here